hopsworks-jwt/src/main/java/io/hops/hopsworks/jwt/filter/JWTFilter.java

```diff
@@ -76,7 +76,7 @@ public void jwtFilter(ContainerRequestContext requestContext) throws IOException
     try {
       Algorithm algorithm = getAlgorithm(jwt);
       JWTVerifier verifier = JWT.require(algorithm)
-          .withIssuer(issuer == null || issuer.isEmpty() ? jwt.getIssuer() : issuer)
+          //.withIssuer(issuer == null || issuer.isEmpty() ? jwt.getIssuer() : issuer)
```

Comment:
This should be re-enabled as https://github.com/logicalclocks/hopsworks/commit/1dbd94248b244c72a102e5aeab7e06124a16003c has been merged to master.